client/app/bundles/course/assessment/submission/components/ScribingView/ScribingToolbar.jsx

```diff
@@ -809,11 +822,7 @@ class ScribingToolbar extends Component {
           </Tooltip>
         </Grid>
         <Grid item xs={1}>
-          <SavingIndicator
-            hasError={this.props.scribing.hasError}
-            isSaved={this.props.scribing.isSaved}
-            isSaving={this.props.scribing.isSaving}
-          />
+          <SavingIndicator savingStatus={this.getSavingStatus()} />
```

Comment:
It’s fine since no one is using scribing. Can just add a feature enhancement for saving indicator to be done in the future. I try not to have too many obsolete components in our codebase

Reply:
I see alright!!